### DISCUSSION TITLE
Is it possible for the franchise collections to combine both TV and Movies

### DISCUSSION THREAD

**Discussion by AaronArgent** [meisnate12/Plex-Meta-Manager#1764] [General]
Is it possible for the franchise collections to combine both TV and Movies?

A couple Examples:
- X Files collection to include: 
   - TV - The X Files
   - Movies - The X Files, The X Files: I Want to Believe

- Plebs Collection to include: 
   - TV - Plebs
   - Movies - Plebs: Soldiers of Rome

- Underbelly Collection to include: 
   - TV - Underbelly, Underbelly NZ - Land Of The Long Green Cloud
   - Movies - Underbelly Files: Tell Them Lucifer Was Here, Underbelly Files: Infiltration

**Comment by chazlarson** [comment]
No.  Plex collections cannot span libraries.  Playlists can.

**Comment by AaronArgent** [comment]
You can manually add TV Shows to Movie Collections and vice versa as below 
![Screenshot 2023-12-07 135040](https://github.com/meisnate12/Plex-Meta-Manager/assets/4347077/75caf5f0-a5c9-40e6-bf95-19c9a2693e88)
![Screenshot 2023-12-07 135236](https://github.com/meisnate12/Plex-Meta-Manager/assets/4347077/4e9ae8df-84cc-4e32-be78-cfefb3d51f13)

**Comment by chazlarson** [comment]
Sure, if you create standard collections with the same name they will show up like that in the ui.  The collections pmm creates are no different.
This doesn't work for smart collections, which a few of the default collections are.
The collections are still in different libraries, though.

**Reply by AaronArgent** [reply]
can the PMM Franchise collection do the above?

**Reply by chazlarson** [reply]
I'm not sure there are any common franchises, but if there are and they're named the same this will happen.

**Reply by chazlarson** [reply]
The Franchise data is taken from TMDB; so it depends on what they have there, but if, for example, TMDB has both Plebs and Plebs: Soldiers of Rome in a collection called "Plebs" then this should happen automatically.
